db/schema.rb

```diff
@@ -0,0 +1,107 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# Note that this schema.rb definition is the authoritative source for your
+# database schema. If you need to create the application database on another
+# system, you should be using db:schema:load, not running all the migrations
+# from scratch. The latter is a flawed and unsustainable approach (the more migrations
+# you'll amass, the slower it'll run and the greater likelihood for issues).
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema.define(version: 2019_07_19_061403) do
+
+  create_table "admins", force: :cascade do |t|
```

Comment:
Nếu tạo ra 2 table là adminds và users, thì khi login em phải tìm kiếm cả 2 table ah
=> chị nghĩ hiện tại em chỉ nên dùng 1 table user, đồng thời thêm field `roles` là dc ấy

-> nếu làm thế thì em tìm hiểu thêm về `enum in rails` nhé. vì field mình nên sử dụng enum đấy

Reply:
login cho user và cho admin riêng ạ.
admin cũng không có quyền đăng nhập vào trang người dùng ạ.